### PR TITLE
Adding my gold calculator

### PR DESCRIPTION
**Pull request by Brennaleker**
For this project I completed the breakfast, lunch and dinner requirements as well as the float requirement from the dessert level

Thank you for taking the time to review my code :)

### CODE REVIEW

**Comment by dougcole** [comment]
starting my review.

File: calculator_clean.rb [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  \tmultiply \t* \n
+  \tdivide \t\t/ \n
+  \tremainder \t% \n
+  \texponent \t^ \n"
+puts user_prompt
+#capturing user input for the operator
+operator = gets.chomp
+
+#if the user did not select one of the options given or mistyped, prompt them to try again
+until ["add", "+", "subtract", "-", "multiply", "*", "divide", "/", "remainder", \
```
Comment (dougcole) [inline calculator_clean.rb:22]: I like the retry with the repeated prompt!

File: calculator_clean.rb [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  answer = number_1 % number_2
+  puts "#{number_1} % #{number_2} = #{answer}"
+
+#if the user typed in "exponent" or "^" the program will give the result of the first number to the second number's power
+when "exponent", "^"
+  answer = number_1 ** number_2
+  puts "#{number_1} ^ #{number_2} = #{answer}"
+
+#if the user did not select one of the options given or mistyped prompt them to try again
+else puts "Please type an operator from the list.\n
```
Comment (dougcole) [inline calculator_clean.rb:102]: line 22 should protect against this else being necessary.

File: calculator_clean.rb [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+#if the user typed in "remainder" or "%" the program will give them the remainder left after the program divides number 1 by number 2
+when "remainder", "%"
+  answer = number_1 % number_2
+  puts "#{number_1} % #{number_2} = #{answer}"
+
+#if the user typed in "exponent" or "^" the program will give the result of the first number to the second number's power
+when "exponent", "^"
+  answer = number_1 ** number_2
+  puts "#{number_1} ^ #{number_2} = #{answer}"
```
Comment (dougcole) [inline calculator_clean.rb:99]: If you make a new variable to store the operator you want to print out you could reduce the duplication in all these puts statements, pulling out something like `puts "#{number_1} #{operator_for_display} #{number_2} = #{answer}"` after the case statement.

**Comment by dougcole** [comment]
Overall this looks great! Let me know if you have any questions on my feedback or about the assignment in general!

**Comment by Brennaleker** [comment]
Hi Doug,
 I will definitely try to store any values that I will be reusing as variables in future projects. Thank you for taking the time to review my code. Your input is very much appreciated.

**Pull request merged**